go/mysql/schema.go

```diff
@@ -129,6 +129,12 @@ order by table_name, ordinal_position`
 	SelectFromViewsTable = `SELECT TABLE_NAME FROM _vt.views WHERE TABLE_NAME IN ::TABLE_NAME`
 
 	SelectAllViews = `SELECT TABLE_NAME, UPDATED_AT FROM _vt.views`
+
+	// FetchUpdatedViews queries fetches information about updated views
+	FetchUpdatedViews = `SELECT TABLE_NAME, VIEW_DEFINITION, CREATE_STATEMENT FROM _vt.views WHERE TABLE_NAME IN ::viewNames`
+
+	// FetchViews queries fetches all views
+	FetchViews = `SELECT TABLE_NAME, VIEW_DEFINITION, CREATE_STATEMENT FROM _vt.views`
```

Comment:
Any particular reason to use all caps for the views queries? Nothing wrong with it, it's just non-standard within Vitess and even this package.

Reply:
there is nothing in particular.
changed queries to lowecase